### DISCUSSION TITLE
跳票红的提示咋改了

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#3561] [提问] [answered]
之前不是说虽然咕咕但也要恰饭吗，咋变这个了（不够明显？）
![image](https://github.com/Hex-Dragon/PCL2/assets/121204926/875f3219-84ad-4ecf-811d-0d783ff604c6)

**Comment by SqrtTan** [comment]
还有这又是啥
![image](https://github.com/Hex-Dragon/PCL2/assets/121204926/53090214-2b31-4d8e-a475-c20c10eef8e4)

**Reply by SqrtTan** [reply]
好了我大概知道原因了
![image](https://github.com/Hex-Dragon/PCL2/assets/121204926/69c07ceb-a1ce-41d5-ad95-abe69efb050f)
（再见砸电脑去了

**Reply by wuliaodexiaoluo** [reply]
6

**Comment by WForst-Breeze** [comment, marked as answer]
此表述于 2.6.15 修改，尽管未在更新日志体现，估计就是为了能把主题获取方式写的更清楚一点而已
